tests/Microsoft.Identity.Test.Unit/PublicApiTests/PublicClientApplicationTestsWithB2C.cs

```diff
@@ -131,6 +131,18 @@ public async Task B2CAcquireTokenWithValidateAuthorityTrueAndRandomAuthorityTest
             }
         }
 
+        [TestMethod]
+        [TestCategory("B2C")]
```

Comment:
Please add a comment mentioning that this is a test against the bug https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3471